### PR TITLE
Fire alarm recolors!

### PR DESCRIPTION
**Pull request by Chada-1**
The fire alarm actually blending in to the station could be seen as an oversight. But it seems that isn't enough to make it a bugfix.
[See here for the recolor ingame](https://forums.aurorastation.org/uploads/monthly_2020_04/Ca5PT0y.gif.127ac1090318cd577e94ba73581b05a0.gif)
[See here for the old fire alarm that was v. difficult to see on the walls.](https://i.imgur.com/W2BoeVZ.png)
This is a PR made in response to this thread: https://forums.aurorastation.org/topic/14263-recolor-the-stations-fire-alarms/?tab=comments#comment-135447

### CODE REVIEW

**Comment by Chada-1** [comment]
!review

**Comment by Chada-1** [comment]
!bugfix

File: html/changelogs/firealarmrecolor.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Added the fire alarm overlay to the above lighting layer and recolored its frame to be red so it's no longer near invisible on the station walls."
```
Comment (NonQueueingMatt) [inline html/changelogs/firealarmrecolor.yml:41]: That's not a bugfix.

**Comment by Chada-1** [comment]
Alright I'm updating the changelog and removing my bugfix comment. 1 minute 👀

File: html/changelogs/firealarmrecolor.yml (hunk 1 above)
Comment (Chada-1) [reply html/changelogs/firealarmrecolor.yml]: Alright that's fixed.

**Pull request merged**